app/serializers/user_serializer.rb

```diff
@@ -0,0 +1,6 @@
+class UserSerializer
```

Comment:
Logan - when we have pairing time will you give me the rundown on serializers?

Reply:
Of course